src/main/java/it/okkam/validation/FiscalCodeValidator.java

```diff
@@ -420,15 +408,15 @@ private static void addNameWithoutDashes(final Map<String, List<String>> comuniM
 
   private static void addToComuniMap(final Map<String, List<String>> comuniMap,
       final int maxComuneNameLength, final String nomeComune, final String codIstat) {
-    if (comuniMap.get(nomeComune) == null) {
+    if (!comuniMap.containsKey(nomeComune)) {
       comuniMap.put(nomeComune, new ArrayList<>());
     } // else nomeComune has multiple codes;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3ACommentedOutCodeLine)